socialhome/users/tests/test_views.py

```diff
@@ -370,18 +386,27 @@ def test_get_json_context(self):
         self.assertEqual(
             view.get_json_context(),
             {
-                "currentBrowsingProfileId": profile.id,
+                "currentBrowsingProfileId": request.user.id,
```

Comment:
Same here?

Reply:
You can also probably remove the `view.followers_count = 3` from this test slightly above line 385.